src/app/profile-edit/profile-edit.page.ts

```diff
@@ -0,0 +1,148 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { LoaderService } from '../services/loader/loader.service';
+import { finalize, catchError } from 'rxjs/operators';
+import { of } from 'rxjs';
+import { ApiBaseService } from '../services/base-api/api-base.service';
+import { ToastService } from '../services/toast/toast.service';
+import urlConfig from 'src/app/config/url.config.json';
+import { FETCH_Profile_FORM } from '../core/constants/formConstant';
+import { MainFormComponent } from 'elevate-dynamic-form';
+
+enum EntityType {
+  State = 'state',
```

Comment:
Why are we hard-coding these entity types? Entity can be anything not specific to location